### DISCUSSION TITLE
ERROR  Cannot restart nuxt:  Cannot set properties of undefined (setting 'nuxtHighcharts')

### DISCUSSION THREAD

**Discussion by CiaranGames** [richardeschloss/nuxt-highcharts#61] [Q&A]
I get this error when I try to put in the chart, I coppied in an example from the https://github.com/richardeschloss/nuxt-highcharts/blob/master/components/charts/BasicChart.vue

nuxt.config.ts

```
import {defineNuxtConfig} from 'nuxt/config'
//
// // https://v3.nuxtjs.org/api/configuration/nuxt.config
export default defineNuxtConfig({
    app: {
        head: {
            title: "Dash",
         }
     },
    modules: [
        'nuxt-highcharts'
    ],
})
```

**Comment by richardeschloss** [comment]
What versions are you using? The master branch contains working code for 3.x

**Comment by CiaranGames** [comment]
I'm currently using:

- "nuxt": "^3.0.0",
- "nuxt3": "^3.0.1-rc.0-27853196.30c4a59",

**Reply by richardeschloss** [reply]
Only "nuxt" is supposed to be installed since the stable version is now nuxt3.

**Reply by CiaranGames** [reply]
I uninstalled "nuxt3" and reinstalled "nuxt" so now I'm on "nuxt": "^3.2.2",

**Reply by CiaranGames** [reply]
Im still getting the error though

**Reply by CiaranGames** [reply]
@richardeschloss Any updates?

**Reply by richardeschloss** [reply]
I'm very busy lately. My advice to checkout this git repo locally, spin it up and then let me know how far you get with that.
